packages/main/src/RadioButton.js

```diff
@@ -70,7 +70,7 @@ const metadata = {
 		 * @defaultvalue false
 		 * @public
 		 */
-		selected: {
+		checked: {
```

Comment:
Determines whether the component is selected or not.

->

**Defines** whether the component is **checked** or not.

Reply:
Done